build/Makefile

```diff
@@ -157,6 +178,24 @@ validate-xml-content: $(GEN_XML_FILES) ## Validate XML files
 	$(MAKE) -C $(OSCAL_CORE_DIR) $(subst $(OSCAL_CORE_DIR)/,,$(OSCAL_COMPLETE_XML_SCHEMA))
 	$(XMLLINT_PATH) --schema $(OSCAL_COMPLETE_XML_SCHEMA) --noout $(GEN_XML_FILES)
 
+
+#
+# Validate XML with oscal-cli
+#
+.PHONY: validate-xml-by-cli
+validate-xml-by-cli: $(OSCAL_CLI_INSTALL_PATH) ## Validate XML files by directory using OSCAL CLI Tool
+	@echo "Validating XML files by directory using OSCAL CLI Tool"
+	@$(OSCAL_CLI_INSTALL_PATH)/bin/$(OSCAL_CLI_BIN) --version
+	@echo "latest oscal-cli version is $(OSCAL_CLI_VERSION)"
```

Comment:
This is only a suggestion , not a blocker. Provide what it is used for validation:
```
@echo "Validating OSCAL content with $(OSCAL_CLI_INSTALL_PATH)/bin/oscal-cli version $(OSCAL_CLI_VERSION)"
```